### ISSUE TITLE
nvcuda.dll is missing from the system

### ISSUE THREAD

**Issue by jmusac** [fireice-uk/xmr-stak#2313]
Downloaded version 2.10 for windows. 
I have issue only on Windows 8. Processors are various pentium version G3XXX and G4XXX G5XXX.
Computers does not have discrete cards like amd or nvidia, only intel integrated.

After running miner and initial setup, the miner chrashes with error: nvcuda.dll is missing from your system.

**Comment by psychocrypt** [comment]
please reinstall cuda and install the full cuda toolkit

**Comment by jmusac** [comment]
I don't have cuda installed. Why is that requirement ? It wasn't requirement in previous version

**Comment by psychocrypt** [comment]
Ohh sry if you have no Nvidia gpus than no cuda is required. could you
please post all output until the crash. to not remove any line.

**Comment by jmusac** [comment]
Here it is:

-------------------------------------------------------------------
[2019-03-11 08:54:44] : Mining coin: cryptonight_r
[2019-03-11 08:54:44] : WARNING: No AMD OpenCL platform found. Possible driver i
ssues or wrong vendor driver.
[2019-03-11 08:54:44] : WARNING: backend AMD (OpenCL) disabled.
[2019-03-11 08:54:44] : NVIDIA: try to load library 'xmrstak_cuda_backend_cuda10
_0'
WARNING: NVIDIA cannot load backend library: xmrstak_cuda_backend_cuda10_0.dll
[2019-03-11 08:54:47] : NVIDIA: try to load library 'xmrstak_cuda_backend_cuda9_
2'
WARNING: NVIDIA cannot load backend library: xmrstak_cuda_backend_cuda9_2.dll
[2019-03-11 08:54:47] : NVIDIA: try to load library 'xmrstak_cuda_backend'
WARNING: NVIDIA cannot load backend library: xmrstak_cuda_backend.dll
[2019-03-11 08:54:52] : WARNING: backend NVIDIA disabled.
[2019-03-11 08:54:52] : Starting 1x thread, affinity: 0.
[2019-03-11 08:54:52] : hwloc: memory pinned
[2019-03-11 08:54:52] : Fast-connecting to 192.168.3.105:8080 pool ...
[2019-03-11 08:54:52] : Pool 192.168.3.105:8080 connected. Logging in...
[2019-03-11 08:54:53] : Difficulty changed. Now: 550.
[2019-03-11 08:54:53] : Pool logged in.
[2019-03-11 08:54:53] : enable cryptonight_r asm 'off' cpu's

**Comment by Spudz76** [comment]
yeah that's what it looks like when there are no CUDAs it continues to run, right?

if not, use `--noNVIDIA --noAMD` command args or compile actual CPU-only exe.

**Comment by laga-bu** [comment]
The program also can be run without Nvidia Gpus. 
I think the problem is  maybe use the Integrated Graphics . I run the program with Desktop ,there is no problem , when I use laptop (Integrated Graphics) to run , it's also have error : nvCuda.dll is missing

**Comment by psychocrypt** [comment]
is the miner crashing? I do not see the error message that nvcuda.dll is
missing like you wrote into the headline of this issue.

**Comment by jmusac** [comment]
Yes, miner is crashing with nvcuda.dll message in separate window.

**Comment by jmusac** [comment]
I'll try to compile for CPU-only and see whether it helps.

**Comment by Spudz76** [comment]
It will definitely help just like `--noNVIDIA` commandline would on regular release.
Both actions block all CUDA DLL loading which would help
